documentation/library_guide/parallel_api/additional_algorithms.rst

```diff
@@ -76,17 +76,26 @@ header.  All algorithms are implemented in the ``oneapi::dpl`` namespace.
     search sequence: [0, 2, 4, 7, 6]
     result sequence: [1, 4, 8, 10, 10]
 
-* ``sort_by_key``: performs a stable key-value sort. The algorithm sorts the sequence's keys according to 
-  a comparioson operator. If no comparator is provided, then the elements are compared with ``operator<``.
-  The sequence's values are permutated according to the sorted sequence's keys. The prerequisite for correct
-  behavior is that the size for both keys sequence and values sequence shall be the same.  
+* ``sort_by_key``: performs a key-value sort. The algorithm sorts a sequence of keys according to
+  a comparison operator. If no comparator is provided, then the elements are compared with ``operator<``.
+  A sequence of values is simultaneously permuted according to the sorted order of keys. For correct behavior
+  both sequences shall be of the same size.  

```

Comment:
As far as we are changing that sentence... Probably, it makes sense to change `shall be` => `muts be` ? We use `muts be` everywhere in the doc.

Reply:
I like `must be` more. Done.